#181414543 Fix PostgresDB Setup

mucyoalexaxel opened the pull request:
#### What does this PR do?

1. It Fixes ```npm run start``` For Barefoot Nomad App.

#### Description of Task to be completed?

1. Fixing ```npm run start``` which was not starting the Barefoot Nomad Server App For Production. 

#### How Should This Be Manually Tested?

1. Check The `Readme` file 

#### What are the relevant pivotal tracker stories?
Pivotal Tracker Story: [https://www.pivotaltracker.com/story/show/181414543](Configure Barefoot Nomad PostgresDB)

rukundo-kevin committed:
Fix: configure ESlint

SergeNm committed:
Setup Unit Test

SergeNm committed:
Setup Unit Test

SergeNm committed:
Setup Unit Test

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

SergeNm committed:
fix .eslintrc

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
fix-PostgresDB_Setup-181414543

mucyoalexaxel committed:
fix-PostgresDB_Setup-181414543

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
# This is a combination of 33 commits.
# This is the 1st commit message:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #2:

fix .eslintrc

# This is the commit message #3:

fix .eslintrc

# This is the commit message #4:

fix .eslintrc

# This is the commit message #5:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #6:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #7:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #8:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #9:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #10:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #11:

Setup Unit Test

# This is the commit message #12:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #13:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #14:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #15:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #16:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #17:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #18:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #19:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #20:

Setup Unit Test

# This is the commit message #21:

fix .eslintrc

# This is the commit message #22:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #23:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #24:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #25:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #26:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #27:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #28:

Setup Unit Test

# This is the commit message #29:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #31:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #32:

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #33:

change .env.example

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

SergeNm reviewed (commented):
The server and database are connecting successfully.

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
# This is a combination of 30 commits.tree 287125b278f8da0b1bf29f48e50cc9e3e6fab07b
parent 0a5aad7212a7d64e2f164de3149685ecdcf00565
author Alex Axel <mucyoalexaxel@gmail.com> 1647095737 +0200
committer Alex Axel <mucyoalexaxel@gmail.com> 1647095737 +0200

ch-setup-API-Swagger-Documentation-181414596

# This is the commit message #30:

fix .eslintrc

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
ch-setup-API-Swagger-Documentation-181414596

mucyoalexaxel committed:
fix-PostgresDB_Setup-181414543

mucyoalexaxel committed:
fix-PostgresDB_Setup-181414543

mucyoalexaxel committed:
Merge branch 'develop' into fix-PostgresDB_Setup#181414543

mucyoalexaxel committed:
fix-PostgresDB_Setup-181414543

mucyoalexaxel committed:
Deleted .DS_Store

mucyoalexaxel committed:
Deleted .DS_Store
